app/misc/authenticate.ts

```diff
@@ -180,9 +180,10 @@ function authenticateUser(callback) {
       });
 
     }, err => {
-      console.log('Error while getting token', err);
+      console.log('ERROR while getting token. ', err);
 	}).catch( err => {
-    console.log(err);
+    let tokenError = err;
```

Comment:
Is `tokenError` used other than initializing it to equal `err` ?

Reply:
Nope. Realized after the fact that I could just use err and it wouldn't cause any problems. Just didn't go back and fix it due to laziness.